x-pack/plugins/cloud_security_posture/public/pages/configurations/configurations.tsx

```diff
@@ -20,9 +20,11 @@ export const Configurations = () => {
   const location = useLocation();
   const dataViewQuery = useLatestFindingsDataView();
   const getSetupStatus = useCspSetupStatusApi();
-
-  const hasFindings = getSetupStatus.data?.status === 'indexed';
-  if (!hasFindings) return <NoFindingsStates />;
+  const hasFindings =
+    getSetupStatus.data?.indicesDetails[0].status === 'not-empty' ||
+    getSetupStatus.data?.kspm.status === 'indexed' ||
+    getSetupStatus.data?.cspm.status === 'indexed';
```

Comment:
Let's create a single function to return all the "auxiliary" status based on the `getSetupStatus.data` response

example:

```js
// Cloud Posture Management Status
getCpmStatus = (cpmStatusData) => {
  // if has findings in any of the integrations.
  const hasFindings =
    getSetupStatus.data?.indicesDetails[0].status === 'not-empty' ||
    getSetupStatus.data?.kspm.status === 'indexed' ||
    getSetupStatus.data?.cspm.status === 'indexed';

  // kspm
  const hasKspmFindings =
    getSetupStatus.data?.kspm?.status === 'indexed' ||
    getSetupStatus.data?.indicesDetails[0].status === 'not-empty';

  // cspm
  const hasCspmFindings =
    getSetupStatus.data?.cspm?.status === 'indexed' ||
    getSetupStatus.data?.indicesDetails[0].status === 'not-empty';

  const isCspmInstalled =
    getSetupStatus.data?.cspm.status !== 'unprivileged' &&
    getSetupStatus.data?.cspm.status !== 'not-installed';

  return {
    hasFindings,
    hasKspmFindings,
    hasCspmFindings,
    isCspmInstalled,
    // ... and so on
  };
};

```

This way we consolidate the logic in a single place instead of all the respective conditions